src/mmw/apps/user/views.py

```diff
@@ -47,6 +47,18 @@ def ajax_login(request):
     return Response(data=response_data, status=status)
 
 
+@decorators.api_view(['GET'])
+@decorators.permission_classes((AllowAny, ))
+def user_logout(request):
+    logout(request)
+
+    if request.is_ajax():
+        response_data = {'guest': True, 'result': 'success'}
+        return Response(data=response_data)
+    else:
+        return render_to_response('user/logout.html')
+
```

Comment:
I like this much better than having to maintain separate urls/views for ajax vs non-ajax requests. We should consider doing the same for "ajaxlogin" in the future.